csrc/codegen.cpp

```diff
@@ -2972,6 +2960,10 @@ class CudaKernelGenerator : private kir::ConstIrVisitor {
           aligned_array_of_regs_.insert(tv);
         }
       }
+      // tv is aligned if alias is aligned
+      if (aligned_array_of_regs_.count(alias_tv) > 0) {
+        aligned_array_of_regs_.insert(tv);
```

Comment:
What is this variable used for? It seems it's only used by `genVariableNameConvertAlinedArray`, which is only used by `generateGroupedGridAllReduceWelfordOuter`. I wonder if this change matters for this PR. Also, it doesn't seem to be used consistently. Am I understanding correctly?

Reply:
Never mind, using it was part of the PR changes.